### PR TITLE
create Employee model with validations

### PR DESCRIPTION
**Pull request by DadaLV**

### CODE REVIEW

File: db/migrate/20230509214343_create_employees.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+class CreateEmployees < ActiveRecord::Migration[7.0]
+  def change
+    create_table :employees do |t|
+      t.integer :id_number
```
Comment (kvasyaa) [inline db/migrate/20230509214343_create_employees.rb:4]: the system creates `id` automatically.

File: test/fixtures/employees.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
```
Comment (kvasyaa) [inline test/fixtures/employees.yml]: use less file

File: test/models/employee_test.rb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
```
Comment (kvasyaa) [inline test/models/employee_test.rb]: use less file

File: app/models/employee.rb [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+  validates :first_name, presence: true, length: {minimum: 2, maximum: 30}
+  validates :middle_name, presence: true, allow_blank: true, length: {minimum: 2, maximum: 30}
+  validates :last_name, presence: true, length: {minimum: 2, maximum: 30}
+  validates :passport_data, presence: true, uniqueness: true, format: { with: /\A[A-Za-z]{2}\d{6}\z/, message: "must have two letters followed by six digits" }
+  validates :date_of_birth, presence: true
+  validates :date_of_birth_not_younger_than_15_years
+  validates :place_of_birth, presence: true, length: {minimum: 2, maximum: 30}
+  validates :home_address, presence: true, length: {minimum: 2, maximum: 30}
+
+  def date_of_birth_not_younger_than_15_years
```
Comment (kvasyaa) [inline app/models/employee.rb:14]: this method can be private

**Review by kvasyaa — CHANGES_REQUESTED** [review]
cannot see the proper validation errors.
Знімок екрана 2023-05-10 о 12 59 20

File: db/migrate/20230509214343_create_employees.rb (hunk 1 above)
Comment (DadaLV) [reply db/migrate/20230509214343_create_employees.rb:4]: so we don't need to write "табельний номер" ? should I delete it? I can't understand

Comment (kvasyaa) [reply db/migrate/20230509214343_create_employees.rb:4]: `id` - will be a unique number

File: app/models/employee.rb [hunk 5]
```diff
@@ -0,0 +1,19 @@
+class Employee < ApplicationRecord
+  validates :id_number, presence: true
+  validates :first_name, presence: true, length: {minimum: 2, maximum: 30}
+  validates :middle_name, presence: true, allow_blank: true, length: {minimum: 2, maximum: 30}
```
Comment (kvasyaa) [inline app/models/employee.rb]: Chose `presence: true` or `allow_black: true`, but not both.

**Review by kvasyaa — CHANGES_REQUESTED** [review]

**Review by kvasyaa — APPROVED** [review]

**Pull request merged**